csharp/scripts/stubs/make_stubs_all.py

```diff
@@ -0,0 +1,36 @@
+import sys
+import os
+import helpers
+import json
+import shutil
+
+print('Script to generate stub files for all C# packages relevant for tests.')
+print('Please extend the `packages` list in this script to add more packages when relevant.')
+print(' Usage: python3 ' + sys.argv[0] + ' ' + '[WORK_DIR=tempDir]')
+
+# List of packages to create stubs for.
+packages = [
+    "Amazon.Lambda.Core",
+    "Amazon.Lambda.APIGatewayEvents",
+    "Dapper",
+    "Newtonsoft.Json",
+    "NHibernate",
+    "ServiceStack",
+    "ServiceStack.OrmLite.SqlServer",
+    "System.Data.SqlClient",
+    "System.Data.SQLite",
+    ]
```

Comment:
Could we avoid hard coding this list by simply reading the folder names in `csharp/ql/test/resources/stubs`?

Reply:
That is an interesting idea!
If time allows it, I will look into it!

Reply:
I realized that it might not be that easy, because we have a flat list in the folder structure, which means we can't differentiate top level nuget packages (which are in the above list) and their dependencies. And most probably we don't want to run the stub generator on the dependencies individually.

Reply:
Yeah, that could cause a problem.